### PR TITLE
Plan Features FAQ: Include eCommerce Plan

### LINKED ISSUE #34721: Plans: no mention of eCommerce plan in the FAQ section
#### Steps o reproduce
1. Starting at URL: /plans and then scroll down
2. The Custom Domain is also part of the eCommerce plan

Screenshot 2019-07-17 at 18 57 36

#### What I expected

The FAQ to be relevant to our newest plans

#### What happened instead

The FAQ is outdated

#### Browser / OS version

All

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

* Includes mention of the eCommerce plan for the Plan Features FAQ

#### Testing instructions

On a WordPress.com site, visit the Plans page and scroll to the bottom, then verify the copy changes.

Screenshot 2019-07-17 at 17 07 28

Fixes #34721

### CODE REVIEW

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I install plugins?' ) }
 				answer={ translate(
-					'Yes! With the WordPress.com Business plan you can search for and install external plugins.' +
+					'Yes! With the WordPress.com Business or eCommerce plan you can search for and install external plugins.' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Yes! With the WordPress.com Business plan you can search for and install external plugins. All plans already come with a custom set of plugins tailored just for them. {{a}}Find out more about plugins{{/a}}.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I upload my own theme?' ) }
 				answer={ translate(
-					"Yes! With the WordPress.com Business plan you can upload any theme you'd like." +
+					"Yes! With the WordPress.com Business or eCommerce plan you can upload any theme you'd like." +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Yes! With the WordPress.com Business plan you can upload any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 3]
```diff
@@ -87,14 +87,14 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
...
-						' when you upgrade to the Personal, Premium, or Business plan.'
+						' when you upgrade to the Personal, Premium, Business or eCommerce plan.'
 				) }
 			/>
 
 			<FAQItem
 				question={ translate( 'Do you offer email accounts?' ) }
 				answer={ translate(
-					'Yes. If you register a new domain with our Personal, Premium, or Business plans, you can' +
+					'Yes. If you register a new domain with our Personal, Premium, Business or eCommerce plans, you can' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Yes. If you register a new domain with our Personal, Premium, or Business plans, you can add Google-powered G Suite. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by dzver** [comment]
LGTM

**Comment by Aurorum** [comment]
Conflicts resolved :)

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 4]
```diff
@@ -73,7 +73,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I install my own theme?' ) }
 				answer={ translate(
-					"Yes! With the WordPress.com Business plan you can install any theme you'd like." +
+					"Yes! With the WordPress.com Business or eCommerce plan you can install any theme you'd like." +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Yes! With the WordPress.com Business plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/34722)

**Comment by lancewillett** [comment]
This looks good to me. @Aurorum Would you mind rebasing? I'll be happy to finish the push.

**Comment by Aurorum** [comment]
Done! :)

**Pull request merged**